x-pack/plugins/security_solution/common/types/timeline/index.ts

```diff
@@ -143,10 +143,15 @@ const SavedFavoriteRuntimeType = runtimeTypes.partial({
 /*
  *  Sort Types
  */
-const SavedSortRuntimeType = runtimeTypes.partial({
+
+const SavedSortObject = runtimeTypes.partial({
   columnId: unionWithNullType(runtimeTypes.string),
   sortDirection: unionWithNullType(runtimeTypes.string),
 });
+const SavedSortRuntimeType = runtimeTypes.union([
```

Comment:
To my understanding it might be
```const SavedSortRuntimeType = runtimeTypes.array(SavedSortObject);```

Reply:
These could be changed to arrays as well:
In `x-pack/plugins/security_solution/server/lib/timeline/default_timeline.ts` line 23
`x-pack/plugins/security_solution/server/lib/timeline/routes/__mocks__/create_timelines.ts` line 178 and 215
`x-pack/plugins/security_solution/server/lib/timeline/routes/__mocks__/request_responses.ts` line 76

Reply:
ah my bad, I understand now, we take both SavedSortObject and SavedSortObject[] here.